### ISSUE TITLE
Obtaining my own critic data

### ISSUE THREAD

**Issue by roynirmal** [AkariAsai/self-rag#25]
First of all congratulations on the impressive work! 
I am looking to extend SELF_RAG for long context tasks and planning to create my own training data. I had 3 questions about obtaining my own critic data.

1. What is the input file? Are the inputs obtained from the tasks themselves? 
2. How do we create the jsonl for the input files? Where do we get those parameters from? (There is no README in `process_data` as mentioned here` data_creation/critic/gpt4_reward/README.md`). Does each critic data need a separate input file?
3. Once I have obtained the critic data for each of the four tokens, how do I combine them into one file for training the critic model?

**Comment by AllenShow** [comment]
YES! There is no README in process_data as mentioned here data_creation/critic/gpt4_reward/README.md

**Comment by AkariAsai** [comment]
Hi, thank you so much for opening an issue! Sorry that the README is not really easy-to-follow. 
I addressed some questions briefly [here](https://github.com/AkariAsai/self-rag/issues/56) and try to improve the documentation in the coming weeks.
